roofit/roofitcore/inc/RooFit/Detail/AnalyticalIntegrals.h

```diff
@@ -173,6 +174,66 @@ inline double chebychevIntegral(double const *coeffs, unsigned int nCoeffs, doub
    return halfrange * sum;
 }
 
+// Clad does not like std::max and std::min so redefined here for simplicity.
+inline double max(double x, double y)
+{
+   return x >= y ? x : y;
+}
+
+inline double min(double x, double y)
+{
+   return x <= y ? x : y;
+}
+
+// The last param should be of type bool but it is not as that causes some issues with Cling for some reason...
+inline double
+poissonIntegral(int code, double mu, double x, double integrandMin, double integrandMax, unsigned int protectNegative)
```

Comment:
```suggestion
poissonIntegralOverX(double mu, double x, double integrandMin, double integrandMax, unsigned int protectNegative)
```
This function implements the `code == 1` case only that integrates over X. Why do you need to take `code` as a parameter then? I would rather rename the function and not take this inactive `code` parameter, which also bloats the generated code

Reply:
Not really, code 2 is implemented by default in that function. There was an `else if` around it but I dropped that because the code can only either be 1 or 2. Or are you saying something else?